<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application::DrawWindow</class>
 <widget class="QWidget" name="DrawWindow">
  <layout class="QGridLayout" name="DrawLayout">
   <item row="3" column="0">
    <layout class="QFormLayout" name="ChooseLayout_1">
     <item row="0" column="0">
      <widget class="QLabel" name="shapeChooseLabel">
       <property name="text">
        <string>Shape</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="shapeChoose"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="brushStyleChooseLabel">
       <property name="text">
        <string>Brush Style</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="brushStyleChoose"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="penWidthChooseLabel">
       <property name="text">
        <string>Pen width</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="penWidthChoose">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QFormLayout" name="ChooseLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="penCapChooseLabel">
       <property name="text">
        <string>Pen cap</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="penCapChoose"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="penStyleChooseLabel">
       <property name="text">
        <string>Pen style</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="penStyleChoose"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="penJoinChooseLabel">
       <property name="text">
        <string>Pen join</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="penJoinChoose"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
